recipes/mkdocs-click/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "mkdocs-click" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 00959d6cfcd9cf15af081f9f9a440027e84bff700fb4c14ba6e5efc0ac75ecf6
+
+build:
+  number: 0
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
  number: 0
```